backend/claim.py

```diff
@@ -1,65 +1,234 @@
 from bs4 import BeautifulSoup
+# from selenium import webdriver
+import exceptions as error
 import requests
 import tokenizer
+from wiki_scraper import wiki
 import sys
 import io
+import re
+import os
+import queue as q
+from config import config
+import uuid
+
+CWD_FOLDER = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This should be part of `config`

Reply:
I don't actually think this one should, because this just adds a file path relative to this file.

Reply:
I don't actually think this one should, because this just adds a file path relative to this file.